src/main/java/frc/robot/commands/BangBangDriveCommand.java

```diff
@@ -0,0 +1,72 @@
+package frc.robot.commands;
+
+import edu.wpi.first.math.Pair;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.DriveConstants;
+import frc.robot.subsystems.Drivetrain;
+import java.awt.geom.Point2D;
+
+public class BangBangDriveCommand extends Command{
+    Drivetrain driv;
+    Pose2d setpoint;
+    int tim;
+    public BangBangDriveCommand(Drivetrain drev, Pose2d setpoint){
+        driv = drev;
```

Comment:
Can reuse parameter name driv instead of drev in the constructor. In WPILib command-based programming especially, it's better to keep parameter name consistent for clarity. Indicates that the parameter is being assigned to the same instance variable. Makes code easier to read and maintain.